### ISSUE TITLE
Select classes

### ISSUE THREAD

**Issue by EnnoMeijers** [netwerk-digitaal-erfgoed/LDWizard-ErfgoedWizard#8]
The 'choose property' dialogue offers only properties and no classes, in certain cases, like pointing to an URI or identifier, this would make sense.

**Comment by wouterbeek** [comment]
@EnnoMeijers IIUC this would be in combination with the ability to create IRI object terms?  (Then there will be exported as instances of the class that is selected for the corresponding column.)

**Comment by EnnoMeijers** [comment]
That's correct or in case of the presence of IRIs in the source data. Note: we are pushing heritage institutes to at least add IRI in their data even if the system is still an old school DBMS.

**Comment by wouterbeek** [comment]
Thanks for the clarification, this would indeed allow for more complex transformations (resulting in 'real' linked data / connections between nodes).
